src/hooks/abrt-hook-ccpp.c

```diff
@@ -836,56 +859,24 @@ int main(int argc, char** argv)
 
     char path[PATH_MAX];
 
-    int src_fd_binary = -1;
-    char *executable = get_executable(pid, setting_SaveBinaryImage ? &src_fd_binary : NULL);
-    if (executable == NULL)
-    {
-        error_msg_not_process_crash(pid_str, NULL, (long unsigned)uid, signal_no,
-                signame, "ignoring (can't read /proc/PID/exe link)");
-
-        xfunc_die();
-    }
-
-    if (strstr(executable, "/abrt-hook-ccpp"))
-    {
-        error_msg_and_die("PID %lu is '%s', not dumping it to avoid recursion",
-                        (long)pid, executable);
-    }
-
-    const char *last_slash = strrchr(executable, '/');
-    if (is_path_ignored(setting_ignored_paths, executable))
-    {
-        error_msg_not_process_crash(pid_str, last_slash + 1, (long unsigned)uid, signal_no,
-                signame, "ignoring (listed in 'IgnoredPaths')");
-
-        return 0;
-    }
-
-    /* dumping core for user, if allowed */
-    if (setting_allowed_users || setting_allowed_groups)
-    {
-        if (setting_allowed_users && is_user_allowed(uid, setting_allowed_users))
-            log_debug("User %lu is listed in 'AllowedUsers'", (long unsigned)uid);
-        else if (setting_allowed_groups && is_user_in_allowed_group(uid, setting_allowed_groups))
-            log_debug("User %lu is member of group listed in 'AllowedGroups'", (long unsigned)uid);
-        else
-        {
-            error_msg_not_process_crash(pid_str, last_slash + 1, (long unsigned)uid, signal_no,
-                signame, "ignoring (not allowed in 'AllowedUsers' nor 'AllowedGroups')");
-
-            xfunc_die();
-        }
-    }
-
-    user_pwd = get_cwd(pid);
-    log_notice("user_pwd:'%s'", user_pwd);
-
     sprintf(path, "/proc/%lu/status", (long)pid);
     char *proc_pid_status = xmalloc_xopen_read_close(path, /*maxsz:*/ NULL);
 
     uid_t fsuid = uid;
     uid_t tmp_fsuid = get_fsuid(proc_pid_status);
+    if (tmp_fsuid < 0)
+    {
+        error_msg_ignore_crash(pid_str, NULL, (long unsigned)uid, signal_no,
+                signame, "parsing error");
```

Comment:
How are users supposed to distinguish between parsing error in of "fsuid" and "fsgid"?
What would you do if you saw this error message in the logs:
````
Process 29288 ( (will_segfault)) of user 1000 killed by SIGSEGV - ignoring (parsing error)
```